From d00ac1002a6a16aca4a57ada0c304eba839a7523 Mon Sep 17 00:00:00 2001
From: scx <scx.mail@gmail.com>
Date: Fri, 15 Feb 2019 21:19:15 +0100
Subject: [PATCH] Enhance AppData file

1. Update release notes
- Fix date for 0.5.0
- Add info about 0.5.1

2. Add screenshot captions

3. Add kudos

4. Add launchable tag

5. Fix indents
---
 data/com.github.parnold-x.nasc.appdata.xml | 23 ++++++++++++++++++----
 1 file changed, 19 insertions(+), 4 deletions(-)

diff --git a/data/com.github.parnold-x.nasc.appdata.xml b/data/com.github.parnold-x.nasc.appdata.xml
index a99ce96..cd087b8 100644
--- a/data/com.github.parnold-x.nasc.appdata.xml
+++ b/data/com.github.parnold-x.nasc.appdata.xml
@@ -12,21 +12,30 @@
   <provides>
     <binary>com.github.parnold-x.nasc</binary>
   </provides>
+  <launchable type="desktop-id">com.github.parnold-x.nasc.desktop</launchable>
   <releases>
-    <release version="0.5.0" date="2018-20-28">
+    <release version="0.5.1" date="2018-11-05">
       <description>
-        <p>Release for Juno</p>
+        <p>Update Debian control file.</p>
+      </description>
+    </release>
+    <release version="0.5.0" date="2018-10-28">
+      <description>
+        <p>Release for Juno.</p>
       </description>
     </release>
   </releases>
   <screenshots>
     <screenshot type="default">
+      <caption>Main view</caption>
       <image>https://raw.githubusercontent.com/parnold-x/nasc/master/Screenshot.png</image>
     </screenshot>
     <screenshot>
+      <caption>Help pane</caption>
       <image>https://raw.githubusercontent.com/parnold-x/nasc/master/Screenshot2.png</image>
     </screenshot>
     <screenshot>
+      <caption>Periodic table</caption>
       <image>https://raw.githubusercontent.com/parnold-x/nasc/master/Screenshot3.png</image>
     </screenshot>
   </screenshots>
@@ -63,9 +72,15 @@
   <url type="homepage">https://github.com/parnold-x/nasc</url>
   <url type="bugtracker">https://github.com/parnold-x/nasc/issues</url>
   <url type="help">https://github.com/parnold-x/nasc/issues</url>
+  <kudos>
+    <kudo>HiDpiIcon</kudo>
+    <kudo>HighContrast</kudo>
+    <kudo>ModernToolkit</kudo>
+    <kudo>UserDocs</kudo>
+  </kudos>
   <custom>
-     <value key="x-appcenter-color-primary">#3278cd</value>
-     <value key="x-appcenter-color-primary-text">rgb(157, 200, 236)</value>
+    <value key="x-appcenter-color-primary">#3278cd</value>
+    <value key="x-appcenter-color-primary-text">rgb(157, 200, 236)</value>
     <value key="x-appcenter-suggested-price">8</value>
   </custom>
 </component>
